WcaOnRails/config/locales/en.yml

```diff
@@ -1665,72 +1665,89 @@ en:
+          '3': "The account you use to register for your first WCA competition will be automatically connected with your WCA ID and profile once the results have been submitted from the competition. Be sure to use an email address that you check often so that any competition communications will reach you when needed."
           #context: Yup, nothing to translate here! Just hit "Use the original version"
-          '3': "%{current_user_status}"
+          '4': "%{current_user_status}"
       'results':
         title: "When are the results of a WCA competition uploaded to the WCA website?"
         #context: Paragraph of the answer to "When are the results of a WCA competition uploaded?"
         content:
           '1': "After a competition has ended, the WCA Delegate has to double-check all score sheets and person data to eliminate possible errors. Once this is done, the results are sent to the WCA Results Team (WRT), which is responsible for uploading competition results and maintaining the WCA database. The WRT finally uploads the results to the WCA database as soon as possible."
           '2': "Depending on the size of the competition and the Delegate's personal schedule, all this can take only a day or up to a week. Please be patient."
           '3': "Additionally, if you have connected your WCA account with your WCA ID (see below), you can enable email notifications about your results submission <a href='%{preferences_edit_path}'>here</a>."
+          '4': "If it is your first competition you will have to wait for the results to be uploaded before you will be assigned a WCA ID."
       '7':
         title: "How do I change my profile picture?"
         #context: Paragraph of the answer to "How do I change my profile picture?"
         content:
           '1': "First, you need to <a href='#how-do-i-connect-my-wca-account-with-my-wca-id'>create a WCA account and connect it to your WCA ID.</a>"
           '2': "After you have connected your WCA account and your WCA ID, go to your <a href='%{profile_edit_path}'>edit profile page</a> to change your profile picture."
       '8':
-        title: "How do I connect my WCA account with my WCA ID?"
+        title: "How do I know what my WCA ID is? How do I connect my WCA account with my WCA ID?"
         #context: Answer to "How do I connect my WCA account ...?"
         content:
           #context: Yup, nothing to translate here! Just hit "Use the original version"
-          '1': "%{current_user_claim}"
+          '1': "After you have competed in your first competition and the results have been published you can go to the competition website to find your results. When you then click your name it will take you to your WCA profile page where your WCA ID will be listed. %{current_user_claim}"
```

Comment:
Sorry I did not notice this before, but the "context" thing above is shown to the translators to provide them with some context: would you mind dropping the line since it's not relevant anymore?
Also the same reasoning for top-level keys applies for nested keys too, so could you please change the `1` to something else?